xml/bp_troubleshooting_status.xml

```diff
@@ -92,13 +92,13 @@
    <listitem>
     <para>
      A manager module has experienced an unexpected error. Typically, this
-     means an unhandled exception was raised from the module’s serve
-     function. The human readable description of the error may be obscurely
-     worded if the exception did not provide a useful description of itself.
-     This health check may indicate a bug. Open a bug report if you think you
-     have encountered a bug. If you believe the error is transient, you may
-     restart your manager daemon(s), or use <command>ceph mgr fail</command> on
-     the active daemon to prompt a failover to another daemon.
+     means an unhandled exception was raised from the module’s serve function.
```

Comment:
```suggestion
     means an unhandled exception was raised from the module’s <literal>serve</literal> function.
```

Reply:
"Serve function" is not a thing, and one of the only places the phrase occurs on the WWW is in our docs. 

The upstream docs contain this reference: https://docs.ceph.com/en/latest/rados/operations/health-checks/

Therefore, I think the term needs to be emphasized or something to indicate we are restricting our mention of a function to one specific one.